internal/view/browser.go

```diff
@@ -562,6 +562,26 @@ func (b *Browser) resourceDelete(selections []string, msg string) {
 			}
 			b.GetTable().DeleteMark(sel)
 		}
+
```

Comment:
I think the main issue here is tho we do check if pf are active on the pod, we fail to delete as the pod fqn does not match the stored pf fqn. I think the fix here is to simply check for the prefix as you doing here in the pk Kill method.
Would this make sense?